documentation/testing/test-plans/20240711_release-test-plan.md

```diff
@@ -0,0 +1,81 @@
+# Release Test Plan
+Release date: 11/07/2024
+
+## Introduction
+Objective: The purpose of this test plan is to outline the approach and scope for testing the bug fixes made today. It will detail the various types of testing that will be conducted as part of release process.  This plan aims to ensure all aspects of the updates are thoroughly tested and meet the required standards before deployment.
+
+### Scope 
+The scope of this testing plan includes thoroughly testing of the fixes made to ensure they function as intended.
```

Comment:
This doesn't read quite write? How about...

`This test plan covers the functional and regression testing to validate all features and bug fixes included in the release.`

Reply:
updated it now.